app/views/index.scala.html

```diff
@@ -196,49 +196,69 @@
             <div class="im-centered3" id="collaborators-title">
                 <div class="row"><br>@Messages("landing.collaborators.title")<br></div>
             </div>
-            <section class="collaborator-images-container">
-                <div class="collaborator-image">
-                    <a id="makeabilitylab-img-link" href="https://makeabilitylab.cs.washington.edu/">
-                        <img src='@routes.Assets.at("assets/MakeabilityLogo.png")' alt='Makeability_Logo'>
-                    </a>
-                </div>
-                <div class="collaborator-image">
-                    <a id="uic-img-link" href="https://idhd.ahs.uic.edu/">
-                        <img src='@routes.Assets.at("assets/uic_logo.png")' alt='UIC_Logo'>
-                    </a>
-                </div>
-                <div class="collaborator-image">
-                    <a id="ligapeatonal-img-link" href="https://ligapeatonal.org/">
-                        <img src='@routes.Assets.at("assets/LigaLogo.png")' alt='LIGA_Logo'>
-                    </a>
-                </div>
-                @if("Columbus" == cityName){
+            @if("Taipei" != cityName && "New Taipei" != cityName && "Keelung" != cityName){
```

Comment:
Could you actually use `cityId` instead of `cityName` since the former is always guaranteed to be unique? That's also my bad for using `cityName` to check for the logos that are specific to Columbus and Amsterdam (could you fix those for me too please? :grin:)